app/assets/javascripts/dashboard_graphs.js

```diff
@@ -138,6 +141,28 @@
     }
   };
 
+
+  ProposalGraph.prototype.refreshSettings = function(data) {
+    return $.ajax({
+      url: this.proposalSettingsUrl,
+      cache: false,
+      success: function(data) {
+        this.parseSettings(data)
+      }.bind(this),
+      data: {
+        group_by: this.groupBy
+      }
+    });
+  };
+
+  ProposalGraph.prototype.parseSettings = function(data) {
+    this.hideSuccessfulProposal = data.find(
+      s => s.key == 'proposals.deactivate_successful_proposal'
```

Comment:
@fv42wid Thank you for this pull request! :smile:. I'll have a look at it next week.

In the meantime, it looks like our dear Hound bot found a parsing error and couldn't analyze code conventions on this file. That's because we don't use ECMAScript 6 features like arrow functions, since we don't use a transpiler and we still try to support some legacy browsers like Internet Explorer 11. Maybe in a few months we change our minds, but till then, would you mind removing modern JavaScript features? Thanks a lot! :pray:

Reply:
@javierm I removed the arrow function and addressed the other warnings I got around the code I changed.